app/views/api/v1/topics/index.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.topics @topics do |t|
```

Comment:
Avoid single characters when iterating a collection, this is not a rule but just a convention in Ruby. I would change `t` for `topic`.